typescript error: Cannot find module '../build/Release/can.node'

I just upgraded to the latest version and am unable to compile my typescript app because of this error:

```
node_modules/socketcan/dist/socketcan.d.ts:1:22 - error TS2307: Cannot find module '../build/Release/can.node' or its corresponding type declarations.

1 import * as can from "../build/Release/can.node";
```

From what I can tell this is because the compiler cannot find the can.d.ts declaration because it is not part of the dist or release folders but remains in src.

Additionally, the exported type for "Message" is not the same as "Message" on RawChannel (which is not exported) so I am not able to properly type `rawChannel.send(message)` calls.

--- a/src/parse_kcd.ts
+++ b/src/parse_kcd.ts
@@ -150,6 +150,7 @@ function makeSignalFromXml(
 	if (Array.isArray(labelset)) {
 		const rawLabels = labelset[0]["Label"];
 		if (rawLabels != undefined) {
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
 			rawLabels.forEach((l: any) => {
 				labelSet[l["$"].value] = l["$"].name;
 			});

--- a/src/socketcan.ts
+++ b/src/socketcan.ts
@@ -22,6 +22,7 @@
 // -----------------------------------------------------------------------------
 // CAN-Object
 
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="./can.d.ts" />
 import * as can from "../build/Release/can.node";
 
@@ -31,6 +32,7 @@ import * as can from "../build/Release/can.node";
  * encoded in CAN messages.
  * @module Signals
  */
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="./can_signals.d.ts" />
 import * as _signals from "../build/Release/can_signals.node";
 

--- a/src/socketcan.ts
+++ b/src/socketcan.ts
@@ -22,6 +22,7 @@
 // -----------------------------------------------------------------------------
 // CAN-Object
 
+/// <reference path="./can.d.ts" />
 import * as can from "../build/Release/can.node";
 
 // -----------------------------------------------------------------------------
@@ -30,6 +31,7 @@ import * as can from "../build/Release/can.node";
  * encoded in CAN messages.
  * @module Signals
  */
+/// <reference path="./can_signals.d.ts" />
 import * as _signals from "../build/Release/can_signals.node";
 
 // import * as _signals from "can_signals";

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -3,7 +3,6 @@
     "target": "ES2021",
     "lib": ["ESNext"],
     "allowJs": false,
-    "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,

